understanding/21/identify-purpose.html

```diff
@@ -9,16 +9,16 @@
 		<h1>Understanding Identify Purpose</h1>
 		<section id="intent">
 			<h2>Intent</h2>
-			<p>The intent of this success criterion is to support personalization and preferences in order for more people to use the web, communicate, and interact with society.</p>
-			<p>Familiar terms and symbols are key for users with a limited vocabulary to being able to use the web. However, what is familiar for some users may not be for other users so programmatically associating user-interface components and icons enables people to load a set of symbols that is appropriate for them.</p>
+			<p>The intent of this Success Criterion is to ensure that the purpose of many elements on a page can be programmatically determined, so that user agents can extract and present that purpose to users using different modalities.</p>
```

Comment:
maybe "specific elements" rather than "many elements"? the latter sounds a bit handwavy